.github/workflows/main.yaml

```diff
@@ -0,0 +1,75 @@
+name: basic tests
+
+on:
+  push:
+    branches:
+      - 'user/**'
+      - 'feature/**'
+      - 'improvement/**'
+      - 'bugfix/**'
+      - 'w/**'
+      - 'q/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildk
+        uses: docker/setup-buildx-action@v1
+        with:
+          buildkitd-flags: --debug
+
+      - name: Login to Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          context: eve/worker/dockerfile
+          file: eve/worker/dockerfile/actions.Dockerfile
+          build-args: BUILDBOT_VERSION=2.0.1
+          tags: "ghcr.io/scality/bert-e:${{ github.sha }}"
```

Comment:
is there cache implicitly implemented into this action?

Reply:
nop, there is no cache on this. The jobs are running on fresh machines, so it can´t be any caches localy

Reply:
I was wondering about using the actions cache feature on this one.

Here's a link a just found about this https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching that's the idea I had in mind.

But let's keep it as a future improvement for now. I was just curious if this could be done or not.